examples/spark/README.md

```diff
@@ -101,20 +106,13 @@ replicationcontroller "spark-master-controller" created
 Then, use the
 [`examples/spark/spark-master-service.yaml`](spark-master-service.yaml) file to
 create a logical service endpoint that Spark workers can use to access the
-Master pod.
+Master pod:
 
 ```console
 $ kubectl create -f examples/spark/spark-master-service.yaml
 service "spark-master" created
 ```
 
-You can then create a service for the Spark Master WebUI:
```

Comment:
I feel like we should present both of these as options. The Spark master UI is a familiar thing that some people might still want access to.